src/popover/container.tsx

```diff
@@ -91,24 +98,26 @@ export default function PopoverContainer({
       width: parseFloat(getComputedStyle(arrow).width),
       height: parseFloat(getComputedStyle(arrow).height),
     };
-    const containingBlock = getContainingBlock(body);
+    const containingBlock = getContainingBlock(popover);
     const containingBlockRect = containingBlock ? containingBlock.getBoundingClientRect() : viewportRect;
 
-    // Round up dimensions (IE11 doesn't handle subpixels too accurately)
-    const bodyRect = body.getBoundingClientRect();
-    const bodyRectCeil = {
-      top: bodyRect.top,
-      left: bodyRect.left,
-      width: Math.ceil(bodyRect.width),
-      height: Math.ceil(bodyRect.height),
+    const bodyBorderWidth = getBorderWidth(body);
```

Comment:
Need to evaluate content size, not popover size, to ensure the actual size of the content is evaluated no matter if the overflow is applied or not. However, we need to consider the container border for the calculations to be precise.